src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -93,6 +96,7 @@ public AcquireTokenForClientParameterBuilder WithAzureRegion(bool autoDetectRegi
 
             CommonParameters.AddApiTelemetryFeature(ApiTelemetryFeature.WithAzureRegion, autoDetectRegion);
             Parameters.AutoDetectRegion = autoDetectRegion;
+            Parameters.RegionToUse = regionToUse;
```

Comment:
if it's an empty string do we still want to set it, as that's the default?

Reply:
changed the default value to null.